prow/metrics/metrics.go

```diff
@@ -26,23 +26,51 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/push"
 	"github.com/sirupsen/logrus"
+	"k8s.io/test-infra/prow/config"
 )
 
-// PushMetrics is meant to run in a goroutine and continuously push
+type configGetter interface {
```

Comment:
I know we do this elsewhere but it doesn't buy us anything really. Use config.Agent directly.

Reply:
Seems useful for your tests. Disregard